pkgs/by-name/ma/marytts/package.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  # Gradle 8 complains about implicit task dependencies when using `installDist`.
+  # See https://github.com/marytts/marytts/issues/1112
+  gradle_7,
+  makeWrapper,
+  jdk,
+  nixosTests,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "marytts";
+  version = "5.2.1-unstable-2024-10-09";
+
+  src = fetchFromGitHub {
+    owner = "marytts";
+    repo = "marytts";
+    rev = "1c2aaa0751b7cef8ae83216dd78b4c61232b3840";
+    hash = "sha256-jGpsD6IwJ67nDLnulBn8DycXCyowssSnDCkQXBIfOH8=";
+  };
+
+  nativeBuildInputs = [
+    gradle_7
+    makeWrapper
+  ];
+
+  mitmCache = gradle_7.fetchDeps {
+    inherit (finalAttrs) pname;
+    data = ./deps.json;
+  };
+
+  # Required for the MITM cache to function
+  __darwinAllowLocalNetworking = true;
+
+  gradleBuildTask = "installDist";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    mv build/install/source/{lib,user-dictionaries} $out
+
+    makeWrapper ${lib.getExe jdk} $out/bin/marytts-server \
+      --add-flags "-cp \"/var/lib/marytts/lib/*:$out/lib/*\" -Dmary.base=/var/lib/marytts" \
```

Comment:
Should we specify `mary.base` through the NixOS option instead? People may want to change that path.

Reply:
Good idea